lib/service/service.go

```diff
@@ -4813,3 +4822,73 @@ func isDebugMode() bool {
 	v, _ := strconv.ParseBool(os.Getenv(teleport.DebugEnvVar))
 	return v
 }
+
+// readHostUUUIDFromStorage tries to read the `host_uuid` value from different storages,
+// depending on where the process is running.
+// If the process is running in a Kubernetes Cluster, this function will attempt
+// to read the `host_uuid` from the Kubernetes Secret. If it does not exist or
+// if it is not running on a Kubernetes cluster the read is done from the local
+// storage: `dataDir/host_uuid`.
+func readHostUUIDFromStorages(ctx context.Context, dataDir string) (string, error) {
```

Comment:
```suggestion
func readHostIDFromStorage(ctx context.Context, dataDir string) (string, error) {
```

I'm not sure if it ever applies to kube pods, but the host ID is not guaranteed to be a UUID, as we're also allowing EC2 instance IDs, so for naming consistency I'd refer to it as "host ID", here and in the following functions.